Source/Relativity.Transfer.SDK.Samples.Repository/FullPathWorkflow/BearerTokenAuthentication.cs

```diff
@@ -12,10 +12,11 @@
 using Relativity.Transfer.SDK.Samples.Core.Configuration;
 using Relativity.Transfer.SDK.Samples.Core.Helpers;
 using Relativity.Transfer.SDK.Samples.Core.Runner;
+using Relativity.Transfer.SDK.Samples.Repository.Common;
 
 namespace Relativity.Transfer.SDK.Samples.Repository.FullPathWorkflow;
 
-[Sample(1, "A bearer token authentication",
+[Sample((int)SampleOrder.BearerTokenAuthentication, "A bearer token authentication",
```

Comment:
Great idea but change input type for `SampleAttribute` class so you don't have to case here. Expose internal readonly property which holds enum as int and execute sort by on this property.

Reply:
Still not changed to enum.